images/image-user/Dockerfile

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM busybox
+ARG ARCH
+FROM $ARCH/busybox
```

Comment:
seems there are no `arm/busybox` and `arm64/busybox` images?

Reply:
@feiskyer : Should we use arm32v7 and arm64v8 ? [Reference](https://github.com/docker-library/official-images#architectures-other-than-amd64)

Reply:
yes we have to use `arm32v7` and `arm64v8`